Clean Swift/Scene.xctemplate/UICollectionViewController/___FILEBASENAME___Interactor.swift

```diff
@@ -14,15 +14,15 @@ protocol ___FILEBASENAMEASIDENTIFIER___InteractorOutput {
 
 class ___FILEBASENAMEASIDENTIFIER___Interactor: ___FILEBASENAMEASIDENTIFIER___ViewControllerOutput {
     var output: ___FILEBASENAMEASIDENTIFIER___InteractorOutput!
-    var worker: ___FILEBASENAMEASIDENTIFIER___Worker!
```

Comment:
I think it's alright to use it that way. property is named worker and it is Worker Protocol type. One store may be conform to many worker protocols.